client/components/Controllers/File/index.tsx

```diff
@@ -1,15 +1 @@
-import * as React from 'react'
-import { StoreProvider, makeStore } from './store'
-import { ControllerProps } from '../../Controllers/Base'
-import Layout from './Layout'
-
-export interface FileProps extends ControllerProps {}
-
-export default function File(props: FileProps) {
-  const store = React.useMemo(() => makeStore(props), Object.values(props))
-  return (
-    <StoreProvider value={store}>
-      <Layout />
-    </StoreProvider>
-  )
-}
+export { default } from './File'
```

Comment:
Hi, @roll Are we following a specific software pattern here? or why wouldn't we export it directly from `./File`?

Reply:
Index creates a "public interface" to the component but with default exports, it looks quite weird I agree. Usually, the established pattern is something like having `index.ts` like:

```
export {Component1} from './comp1'
export {Component2} from './comp2'
```

Generally speaking exact way of using it (via index or components) is not really important on that low level